packages/lexical-editor/src/components/Editor/RichTextEditor.tsx

```diff
@@ -49,8 +60,13 @@ const BaseRichTextEditor: React.FC<RichTextEditorProps> = ({
         }
     };
 
+    const sizeStyle = {
```

Comment:
@Pavel910 I directly assigned the styles to the variable without calling any method.